pkg/server/handler.go

```diff
@@ -289,7 +289,7 @@ func processResourceIdentity(req *http.Request, requestInfo *request.RequestInfo
 	return req, nil
 }
 
-func mergeCRDsIntoCoreGroup(crdLister kcp.ClusterAwareCRDLister, crdHandler, coreHandler func(res http.ResponseWriter, req *http.Request)) restful.FilterFunction {
+func mergeCRDsIntoCoreGroup(crdLister kcp.ClusterAwareCRDClusterLister, crdHandler, coreHandler func(res http.ResponseWriter, req *http.Request)) restful.FilterFunction {
```

Comment:
`ClusterAwareCRDLister` name wasn't good enough?

Reply:
The pattern is `ClusterLister` has a `List()` [across clusters] and a `Cluster()`, and the `Cluster()` returns a type that can `List()` or `Get()`